src/types/onyx/UserMetadata.ts

```diff
@@ -17,6 +17,9 @@ type UserMetadata = {
 
     /** Type of environment the user is using (staging or production) */
     environment?: string;
+
+    /** Information if user dismissed the try new dot popup before. Can be "empty", true or false */
+    tryNewDotDismissed?: string | boolean;
```

Comment:
Given that the string has to be a specific value, I think you can specify that in the type. 
```suggestion
    tryNewDotDismissed?: 'empty' | boolean;
```